app/models/scrape_result_search_params.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class ScrapeResultSearchParams
```

Comment:
instead of putting this in the `app/models` we can put it into the `params` folder, like `app/params/..` what do you @Goose97

Reply:
Agree. Fixed in 063c7b651f0d62d074ba56a7287bb5e20886a1ff